src/inmanta/deploy/scheduler.py

```diff
@@ -243,6 +244,8 @@ def __init__(
         self.executor_manager = executor_manager
         self._state_update_delegate = ToDbUpdateManager(client, environment)
 
+        self._timer_manager = inmanta.deploy.timers.TimerManager(self)
```

Comment:
I don't like passing `self` here. Only do this to be able to schedule global repair/deploys from the timer manager. Is there a cleaner alternative ?